### DISCUSSION TITLE
Showing FluentAppBarItem in a disabled state

### DISCUSSION THREAD

**Discussion by JasonWeise** [microsoft/fluentui-blazor#2343] [Q&A]
Hi all

I have a FluentAppBar with a number of menu items and I would like to show some FluentAppBarItem's in a disabled state but looking through the docs, it doesn't seem that the FluentAppBarItem has a "Disabled" or "Enabled" prop.

Is there any way this can be achieved natively with the FluentAppBarItem component or do we have to do a work around like a conditional rendering of just and icon in place of the FluentAppBarItem.

Seems like this would be a common need but maybe I am wrong.

Thanks in advance!

**Comment by vnbaaij** [comment]
Hi,

Think that would make for a good addition!

For now we don't have anything in the component for that so conditional rendering is the way to go.
